interfaces/udev/backend.go

```diff
@@ -74,7 +75,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 //
 // If the method fails it should be re-tried (with a sensible strategy) by the caller.
 func (b *Backend) Remove(snapName string) error {
-	glob := fmt.Sprintf("70-%s.rules", interfaces.SecurityTagGlob(snapName))
+	glob := fmt.Sprintf("70-%s.rules", snap.SecurityTag(snapName))
 	return ensureDirState(dirs.SnapUdevRulesDir, glob, nil, snapName)
```

Comment:
Same as above. Note that if this is all just a single file, ensureDirState is a huge red-herring. This would all be much simpler.